domain-models-runtime/src/main/java/org/folio/rest/persist/Criteria/Criterion.java

```diff
@@ -231,7 +243,12 @@ private static Criterion processQueryIntern(Iterator<JsonNode> iter, Criterion c
     return cc;
   }
 
-  public String selects2String() {
+  /**
+   * @return SELECT clause - without SELECT - possibly empty string
+   * @deprecated not used by RMB itself anymore and should not be publicly used
+   */
+  @Deprecated
+  public String selects2String1() {
```

Comment:
Either keep the name and make it deprecated, or completely remove it.
I suggest to remove it because this is part of a breaking change and not used: https://github.com/search?q=org%3Afolio-org+selects2String&type=Code

Reply:
Sure.  Removal is the way to go . And sorry about that rename that I missed.